src/content/polyfill.ts

```diff
@@ -1,5 +1,10 @@
 import type { MonetizationEventPayload } from '@/shared/messages';
 (function () {
+  if (document.createElement('link').relList.supports('monetization')) {
+    // console.log('already patched');
```

Comment:
We can remove this log.

Reply:
Ok, but I'll add the comment with same message. So we know what we're doing here.